### PR TITLE
Adding definitions for specific devrel terms

### PR DESCRIPTION
**Pull request by aspleenic**
DevRel specific terms updated include 'Engaged Event Attendee' and 'Talk'

### CODE REVIEW

File: operations/messaging-and-math/developer-relations.md [hunk 1]
```diff
@@ -14,6 +14,14 @@ Some activities that fall under formal Developer Relations:
 
 Please reach out to the [Community Team](mailto:community@mattermost.com) if you have any questions or would like assistance for any activities in their milieu.
 
+## Definitions
+Event Engaged Attendee
```
Comment (jasonblais) [inline operations/messaging-and-math/developer-relations.md]:
```suggestion
* Event Engaged Attendee
```

File: operations/messaging-and-math/developer-relations.md [hunk 2]
```diff
@@ -14,6 +14,14 @@ Some activities that fall under formal Developer Relations:
...
+	* Virtual - any attendee who attends a talk or visits the booth interactively. This can include volunteering their information directly or having it provided by the organizers post-event.
+	* In-person - any attendee who attends a talk, offers their information at a booth or other station, interacts with any Mattermost representative with intent on further examining Mattermost. Interactions to simply look at Mattermost or to discuss philosophies of Mattermost may notbe considered "engaged" - this can be left to the respresentatives discretion.
+	
+Talk
```
Comment (jasonblais) [inline operations/messaging-and-math/developer-relations.md]:
```suggestion
* Talk
```

**Review by jasonblais — COMMENTED** [review]
@aspleenic with this definition, was our engaged event attendee high for WebSummit, because so many came to attend your talk?

https://docs.google.com/document/d/1O-RphVBM10dxd9u26qwNP3TNr-8lDesY_nsVbMIsEDs/edit?ts=6011adcd#bookmark=id.39k9w6jtl7xn

File: operations/messaging-and-math/developer-relations.md (hunk 1 above)
Comment (aspleenic) [reply operations/messaging-and-math/developer-relations.md]: @jasonblais - yes - with the conference attended by about 50k people, my talk was an average size (2500) for the stage/track and topic

**Comment by jasonblais** [comment]
Thanks!

**Review by jasonblais — APPROVED** [review]

**Pull request merged**